setup.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.5.0
+current_version = 0.6.0
```

Comment:
I would say here that we should directly upgrade the Major version.

Reply:
After testing on my local I believe it is fine as it is. It does not break backwards compatibility